### DISCUSSION TITLE
blog/deploy-aspnet-core-web-api-to-amazon-ecs/

### DISCUSSION THREAD

**Discussion by giscus** [codewithmukesh/discussions#21] [General]
# blog/deploy-aspnet-core-web-api-to-amazon-ecs/

In this article, we will be looking into How to Deploy ASP.NET Core Web API to Amazon ECS, aka Amazon Elastic Container Services. So, with this we will cover tons of topics including ASP.NET Core Web API with MongoDB (this will be our sample application, just to demonstrate the usage of multiple docker containers), Dockerizing the Application, Pushing Docker Images to ECR (Elastic Container Registry), Creating Amazon ECS Services and Task Definitions, Port Mappings, Working with VPC and so much more.

https://codewithmukesh.com/blog/deploy-aspnet-core-web-api-to-amazon-ecs/

**Comment by rchowtakuri** [comment]
Great walkthrough

**Comment by perucomsultores** [comment]
Could you please add an extra step to implement Https, thanks a lot!
